### DISCUSSION TITLE
How to capture events in ui.notification_bar?

### DISCUSSION THREAD

**Discussion by Nayananga** [h2oai/wave#1381] [Q&A]
I have set up a notification_bar in my app. I call this function when there is an error happened

```py
def get_popup_notification_bar(bar_type: str, message: str) -> NotificationBar:
    return ui.notification_bar(
            text=message,
            type=bar_type,
            position='top-right',
            events=["dismissed"]
    )
```

Can I set it to none when it is dismissed after timeout? because it keeps a popup for every other page. (I assume it triggers a dismissed event when it disappears as same as side_panel and dialogs)
@mturoci

**Comment by mturoci** [comment]
This event is not currently fired, but would be good if did. Thanks for the suggestion @Nayananga! Tracked in #1386.

**Reply by Nayananga** [reply]
How I can handle events for a notifiaction_bar? I couldn't find an example in documentation

**Reply by mturoci** [reply]
There are not events currently fired.
